src/main/scala/de/heikoseeberger/sbtheader/CommentStyle.scala

```diff
@@ -128,3 +128,7 @@ final class CommentBlockCreator(blockPrefix: String, linePrefix: String, blockSu
   def apply(text: String): String =
     blockPrefix + newLine + lineCommentCreator(text) + newLine + blockSuffix
 }
+
+object NoneCommentCreator extends CommentCreator {
+  override def apply(text: String): String = text
```

Comment:
Please remove type annotation for overridden method.

Reply:
By type annotation you mean return type? `: String`?

Reply:
Yes.

Reply:
Okay, I will. But is it some special Scala code style? What is the purpose?

Reply:
For overridden members it's unnecessary. And yes, it's a special code style: mine ;-)